### DISCUSSION TITLE
타운홀 스페이스 접근이 안됩니다.

### DISCUSSION THREAD

**Discussion by IronKoh** [metamong-friends/metamong-friends.github.io#39] [제보하기]
E83B1213D6533C22 코드로 스페이스 가입하였는데
Change Space로 이동하거나 ProjectView 스페이스에서 이동을 통해 접근을 하려고 하면
계속 캐릭터 선택 창으로 튕겨나옵니다.

**Comment by ACT-TJ** [comment]
안녕하세요!!

저희가 릴리즈 테스트를 진행하며 해당 스페이스 접속을 확인하는 과정에서 잠시 접속이 되지않았던 것으로
판단됩니다. 불편드려 죄송합니다ㅠ. 혹시 금일도 접속이 되지않는다면 다시 한 번 말씀주시면 다른 원인에 대해서도
파악해 보도록 하겠습니다.

감사합니다.
